### ISSUE TITLE
go tests fail due to exceeding default timeout

### ISSUE THREAD

**Issue by ben-st** [Azure-Samples/azure-sdk-for-go-samples#207]
Thank you for your contribution and feedback! Help us review faster by providing the following information:

## This issue is a: (mark with an `x`)

- [X] bug report
- [ ] request for new sample
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)

## Steps to reproduce:
run go test within containerservice with location westeurope

## Errors and log messages:

``` 
*** Test killed with quit: ran too long (10m0s).
exit status 2
FAIL	github.com/Azure/azure-sdk-for-go-samples/containerservice	600.014s
```

## Expected behavior:
go tests run as described in README

## OS and Go versions:
go version go1.10.3 darwin/amd64

## Further info:
successfull running tests can be archived with a timeout longer than 10 minutes.

should this behaviour be documented in the README tests section?
Some tests my fail due to exceeding timeout e.g. containerservice tests, please provide a longer timeout such as: `go test -timeout 15m` 

my last test took approx. 12 min

``` 
╰─$ go test -timeout 20m 
2018/07/22 16:13:17 creating resource group 'azure-samples-go-CreateAKS-rwV3n' on location: westeurope
2018/07/22 16:25:21 created AKS cluster
2018/07/22 16:25:21 retrieved AKS cluster
2018/07/22 16:25:21 deleted AKS cluster
2018/07/22 16:25:21 deleting resources
PASS
ok  	github.com/Azure/azure-sdk-for-go-samples/containerservice	724.634s
```

**Comment by joshgav** [comment]
Thank you for reporting this! We're working on fixing the test system and I included the timeout configuration in #203.

**Comment by ben-st** [comment]
nice :)
